Adding permission for resend-code endpoint at toml level (#4733)
wso2/docs-is: en/identity-server/5.11.0/docs/develop/using-the-self-sign-up-rest-apis.md

!!! info "Related Links" 
    For information on self-registration via the UI instead, see [Self-Registration and Account Confirmation](../../learn/self-registration-and-account-confirmation).

## Enhance the default permission for the resend-code endpoint.

> Update the default permission for the resend-code endpoint in the Self-Sign-up API to utilize identity management permissions prior to production deployment.

Reviewer: ```suggestion
The [resend-code endpoint](https://api-docs.wso2.com/apidocs/is/is511/selfregister-v5.11.0/#!/operations#SelfRegister#meResendCodePost) of the self sign-up rest APIs is used to resend the confirmation code to an authenticated user. While no scopes are required to invoke this API by default, we recommend limiting permissions to this endpoint using scopes prior to production deployment.

To do so, add the following configurations to the `deployment.toml` file
```

Author: Updated with 4c04088b7637768f658b714e2b8494dcdf6e4ae6

> Add the following properties to the `deployment.toml` file.

Reviewer: ```suggestion
```

Author: Updated with 4c04088b7637768f658b714e2b8494dcdf6e4ae6

!!! info "Related Links" 
    For information on self-registration via the UI instead, see [Self-Registration and Account Confirmation](../../learn/self-registration-and-account-confirmation).

## Enhance the default permission for the resend-code endpoint.

The [resend-code endpoint](https://api-docs.wso2.com/apidocs/is/is511/selfregister-v5.11.0/#!/operations#SelfRegister#meResendCodePost) of the self sign-up rest APIs is used to resend the confirmation code to an authenticated user. While no scopes are required to invoke this API by default, we recommend limiting permissions to this endpoint using scopes, prior to production deployment.

To do so, add the following configurations to the `deployment.toml` file

> Update the default permission for the resend-code endpoint in the Self-Sign-up API to utilize identity management permissions prior to production deployment.

Reviewer: ```suggestion
```

> 

Reviewer: ```suggestion
```

> Add the following properties to the `deployment.toml` file.

Reviewer: ```suggestion
```
